src/Microsoft.FeatureManagement/StatusOverride.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+namespace Microsoft.FeatureManagement
+{
+    /// <summary>
+    /// Provides the capability to override whether a feature is considered enabled or disabled when a variant is assigned.
+    /// </summary>
+    public enum StatusOverride
+    {
+        /// <summary>
+        /// Does not affect the feature state.
+        /// </summary>
+        None,
```

Comment:
Could you add empty lines between enum values? Just like what we've done in VariantAssignmentReason.cs